registration/templates/privacy_policy.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
```

Comment:
i18n was loaded here but not used. The actual text could be translated via `translate` or `blocktranslate`.

Reply:
[692300a](https://github.com/gocept/ok_tools/pull/13/commits/692300ad84ba1d788c9c901db6632c8f2869ad8f)